modules/apps/callbacks/types/export_test.go

```diff
@@ -10,11 +13,17 @@ import (
 
 // GetCallbackData is a wrapper around getCallbackData to allow the function to be directly called in tests.
 func GetCallbackData(
+	ctx sdk.Context,
 	packetDataUnmarshaler porttypes.PacketDataUnmarshaler,
-	packetData []byte, srcPortID string, remainingGas,
+	packet channeltypes.Packet,
+	remainingGas,
 	maxGas uint64, callbackKey string,
 ) (CallbackData, error) {
-	return getCallbackData(packetDataUnmarshaler, packetData, srcPortID, remainingGas, maxGas, callbackKey)
+	// TODO(jim): Probably just refactor single occurrence of this in tests
```

Comment:
not sure I follow why this is supposed to change, doesn't the passing of `callbackKey` to `getCallbackData` already handle the dest vs src callback functions?

Also typically we just create these exported fns in the test files to have access to private functions, by adding logic inside we're kinda shifting that a bit.